CONTRIBUTING.md

```diff
@@ -72,64 +72,55 @@ with [RFC 39](https://github.com/QuiltMC/rfcs/blob/master/structure/0039-pr-poli
 To avoid conflicts and make the process faster, the FCP may be ended early when a new Minecraft version is released and
 QM is updated.
 
-### `T: new mappings`
+### `T: new`
 
 **Description**: Used for pull requests that focus mainly on adding new mappings.
 
 **Required Approvals**: 2
-- At least 1 approval must come directly from a Mappings team member, and the other one can come from a Mappings
-Triage member or another Mappings Team member.
+- At least 1 approval must come directly from a Mappings team member.
 
-**Final Comment Period**:
-- Snapshots: 1 day
-- Releases: 2 days
+**Final Comment Period**: 2 days (1 day for snapshots if the PR has 3+ approvals)
 
 **Special cases**:
-- If the PR adds new mappings needed by QSL, the Final Comment Period is shortened to 12 hours.
+- If the PR adds new mappings needed by QSL, the Final Comment Period is shortened to 12 hours, and it can be skipped if it receives 3 approvals or more.
 
 ### `T: refactor`
 
 **Description**: Used for pull request that focus mainly on changing the names of existing mappings.
 
 **Required Approvals**: 3
-- At least 2 approvals must come directly from a Mappings team member, and the other one can come from a Mappings
-Triage member or another Mappings Team member.
+- At least 2 approvals must come directly from a Mappings team member.
 
 **Final Comment Period**:
-- Snapshots: 3 days
-- Releases: 6 days
+- `T: small` and below: 2 days
+- `T: medium`: 3 days
+- `T: large` and above: 2 days
```

Comment:
```suggestion
- `T: large` and above: 4 days
```
I assume this is a typo based on [this](https://github.com/QuiltMC/quilt-mappings/pull/437#discussion_r1210912642) conversation?

Reply:
oh whoops